readthedocs/redirects/migrations/0002_auto_20180217_1121.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2018-02-17 11:21
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I think this migration isn't really necessary here and the other one I'm not sure